### PR TITLE
product card

### PR DESCRIPTION
**Pull request by andrii-petruk**
- [DEMO LINK]( https://andrii-petruk.github.io/layout_product-cards/ )
- [TEST REPORT LINK]( https://andrii-petruk.github.io/layout_product-cards/report/html_report/ )

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
+@import url(reset.css);
+
+/* @import url(card.css); */
```
Comment (andrii-petruk) [inline src/style.css:3]: When importing other css files, an error occurs...

Знімок екрана 2023-04-23 100908

File: src/card.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```
Comment (OleziO) [inline src/card.css:4]: Move to * selector and remove it from all other places.

File: src/card.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  background: var(--color-blue);
+  border: none;
+  border-radius: 5px;
+}
+
+.card__link {
+  display: block;
+  width: 100%;
+  height: 100%;
+  font-family: "Roboto", sans-serif;
```
Comment (OleziO) [inline src/card.css:80]: Move to html selector and remove it from all other places.

**Review by OleziO — CHANGES_REQUESTED** [review]

**Review by loralevitska — APPROVED** [review]
Good job!

**Pull request closed without merging**